example/src/pages/ModalPage.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+import GhostLogo from '../svg/ghost-logo.svg';
+import {LoginScreen} from '../svg';
+
+import { Button,
+  useModal,
+  ConfirmationModal,
+  resetButtonStyles,
+  useNotification,
+} from 'scorer-ui-kit';
+import { INotificationProps } from '../../../dist/Alerts/atom/Notification';
+
+const widthDesk = 480;
+
+const Container = styled.div`
+  margin: 100px 20px 20px 20px;
+  display: grid;
+  grid-template-columns: 200px;
+  row-gap: 15px;
+`;
+
+const LogoBackground = styled.img`
+  position: absolute;
+  top: -545px;
+  left: -570px;
+  width: 1076px;
+  height: auto;
+  object-fit: contain;
+  mix-blend-mode: overlay;
+`;
+
+const Logo = styled(LoginScreen)`
+  width: 300px;
+  object-fit: contain;
+  position: absolute;
+  top: 50%;
+  left: 65%;
+  transform: translate(-50%, -50%);
+`;
+
+const StyledButton = styled.button`
+  ${resetButtonStyles};
+  color: hsl(0, 0%, 65%);
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: italic;
+  line-height: 1.79;
+  color: hsl(0, 0%, 65%);
+  display: block;
+`;
+
+const ButtonGroup = styled.div`
+  text-align: center;
+  margin-top: 10px;
+  ${StyledButton} {
+    margin: 15px auto 0 auto;
+  }
+`;
+
+const CardCover = styled.div`
+    height: 210px;
+    border-radius: 5px 5px 0 0;
+    position: relative;
+    z-index:99;
+    background-image: linear-gradient(114deg, hsl(250, 60%, 62%), hsl(0, 46%, 54%));
+    overflow: hidden;
+  `;
+
+const CardTitle = styled.div`
+  font-size: 20px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: hsl(210, 6%, 47%);
+`;
+
+const CardContent = styled.div`
+  padding: 30px 40px;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.79;
+  letter-spacing: normal;
+  color: hsl(200, 1%, 49%);
+  p {
+    margin: 1.4rem 0
+  }
+`;
+
+const CardModal = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const ModalPage: React.FC = () => {
+
+  // This is how I imagine that the hook is going to end
+  //const {setIsOpen, updateCloseText, closeCallback, updateContent} = useModal();
```

Comment:
Is this  still relevant or are those extra concepts achievable with what you have implemented? Perhaps we can branch these out to their own tasks for future improvement if they are still to do, they sound like they could be worthwhile.

Reply:
Sorry this were just notes. I'll delete it.

Reply:
All good! Those notes come in useful during these reviews.